### DISCUSSION TITLE
Welcome to kitchenowl Discussions!

### DISCUSSION THREAD

**Discussion by TomBursch** [TomBursch/kitchenowl#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by StanDaMan0505** [comment]
Hi, would love to test it on my Raspberry.. But there is no ARM Version so far. Any plans on adding it?

**Reply by TomBursch** [reply]
I created an issue for it. I'll definitely add it
